crates/noirc_evaluator/src/environment.rs

```diff
@@ -47,14 +47,37 @@ impl Environment {
         self.env.end_scope();
     }
 
-    pub fn store(&mut self, name: String, object: Object) {
+    pub fn store(&mut self, name: String, object: Object, is_global: bool) {
```

Comment:
I had added functionality as I tried to get global consts working in the normal evaluator which required making changes to account for the global scope when storing and fetching objects. However, the evaluator kept error'ing out with a message similar to `cannot constrain two constants`. This seemed to happen for when I compiled without global constants using the normal evaluator as well. I can get rid of this logic for clarity, and leave the normal `evaluate_main` to not use global constants as it is right now. Global constants work with `evaluate_main_alt` and `igen` so I held off on further debugging consts with the normal evaluator.